packages/slate-editor/src/modules/editor-v4-rich-formatting/components/BlockQuoteElement/BlockQuoteElement.tsx

```diff
@@ -19,15 +21,35 @@ export const BlockQuoteElement: FunctionComponent<Props> = ({
     className,
     element,
     ...props
-}) => (
-    <blockquote
-        {...attributes}
-        {...props}
-        className={classNames('editor-v4-block-quote-element', className)}
-        data-slate-type={ElementType.BLOCK_QUOTE}
-        data-slate-value={JSON.stringify(element)}
-        style={{ textAlign: element.align }}
-    >
-        {children}
-    </blockquote>
-);
+}) => {
+    const align = element.align ?? Alignment.LEFT;
+    const isEmpty = Node.string(element).length === 0;
+
+    return (
+        <blockquote
+            {...attributes}
+            {...props}
+            className={classNames('editor-v4-block-quote-element', className, {
+                ['editor-v4-block-quote-element--alignLeft']: align === Alignment.LEFT,
+                ['editor-v4-block-quote-element--alignCenter']: align === Alignment.CENTER,
+                ['editor-v4-block-quote-element--alignRight']: align === Alignment.RIGHT,
+            })}
+            data-slate-type={ElementType.BLOCK_QUOTE}
+            data-slate-value={JSON.stringify(element)}
+        >
+            <p
+                data-placeholder={isEmpty ? 'Quote' : undefined}
+                className={classNames('editor-v4-block-quote-element__paragraph', className, {
+                    ['editor-v4-block-quote-element__paragraph--alignLeft']:
+                        align === Alignment.LEFT,
+                    ['editor-v4-block-quote-element__paragraph--alignCenter']:
+                        align === Alignment.CENTER,
+                    ['editor-v4-block-quote-element__paragraph--alignRight']:
+                        align === Alignment.RIGHT,
```

Comment:
It's really worth converting `BlockQuoteelement.scss` to an SCSS module. Then the selectors can be much shorter (i.e. less visual noise).